packages/core/http/core-http-server-mocks/src/test_utils.ts

```diff
@@ -105,9 +111,9 @@ export const createCoreContext = (overrides: Partial<CoreContext> = {}): CoreCon
 });
 
 /**
- * Creates a concrete HttpServer with a mocked context.
+ * Creates a concrete HttpService with a mocked context.
  */
-export const createHttpServer = ({
+export const createHttpService = ({
```

Comment:
Decided to give this a more fitting name, did not rename all variables where this is assigned, but the func name seemed misleading.